gti.c

```diff
@@ -96,6 +100,16 @@ int main(int argc, char **argv)
     return 1;
 }
 
+/*return true if push command found*/
+int checkPushCommand(int argc, char** argv) {
+  for (int i = 1; i < argc; i++) {
+    if (argv[i][0] == '-') continue;
+    if (!strcmp(argv[i], "push")) return 1;
+    break;
+  }
+  return 0;
+}
```

Comment:
So you're checking/strcmp'ing this every frame - would be nicer to do this only once at the top of main() and set a variable. Also, remove the `(void) argc;` cast (line 75), that's just to silence the "argument unused" warning, and you're using it now :)

code style nitpicking:
* I'm using 4 spaces for indentation
* the `{` after the function definition should be on a new line
* use snake_case names, not camelCase
* move the `**` to the variable (`char **argv`, not `char** argv`)